core/src/main/java/org/testcontainers/images/builder/traits/FilesTrait.java

```diff
@@ -19,31 +16,7 @@ default SELF withFileFromFile(String path, File file) {
     }
 
     default SELF withFileFromPath(String path, Path filePath) {
-        return ((SELF) this).withFileFromTransferable(path, new Transferable() {
-
-            @Override
-            public long getSize() {
-                try {
-                    return Files.size(filePath);
-                } catch (IOException e) {
-                    throw new RuntimeException("Can't get size from " + filePath, e);
-                }
-            }
-
-            @Override
-            public int getFileMode() {
-                return DEFAULT_FILE_MODE | (Files.isExecutable(filePath) ? 0755 : 0);
```

Comment:
Yep - this is important. My implementation is [in MountableFile](https://github.com/testcontainers/testcontainers-java/pull/313/files/1b5f253601cc29497468efbce1e63d173b92c3de#diff-9c787192a8c760a6bde2b6851ee74105R273) - do you see any issues with it? I've not yet tested this on Windows, which is something I need to do.